### DISCUSSION TITLE
Orca issueing bad command?

### DISCUSSION THREAD

**Discussion by Rickthebig** [OrcaSlicer/OrcaSlicer#4854] [General]
I found this im the eventlog file of my Duet2 board:
Error: Bad command: EXCLUDE_OBJECT_DEFINE NAME=Orca-PA-Calibration-Test CENTER=0,0 POLYGON=[[25,25],[285,25],[285,285],[25,285],[25,25]]

Is this an Orca issue or is this my humble incompetence?

**Comment by 4eIIIuP** [comment]
Here you need to either enable support for this command in your board or disable EXCLUDE, find in the model settings -> "Other" -> "Exclude model" and uncheck the box
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/9b8bf32f-5a9a-4b18-b71b-64ff88a9ccfb)

**Reply by Rickthebig** [reply]
Let me disagree. The output
"EXCLUDE_OBJECT_DEFINE NAME=..."
is meant to be evaluated in Klipper, but I am using a printer which is RRF based.
In RepRap, there is the M486 command to manage object canceling.

So this is wrong. It is a bug.

**Reply by 4eIIIuP** [reply]
Selected Klipper
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/ec44dc3f-1476-4d4c-9eed-e41167692358)

Selected RepRapFirmware
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/ca73a50a-94dc-4a59-977a-45c0f53827ca)
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/7ce5d3fe-352b-4986-b87a-af9a61e3bf4f)

Version of orca slicer: 2.0.0
As you can see from the G code, everything works well
